9-regular-expressions/06-regexp-boundary/1-find-time-hh-mm/task.md

```diff
@@ -1,9 +1,9 @@
-# Find the time
+# Encontre o horário
 
-The time has a format: `hours:minutes`. Both hours and minutes has two digits, like `09:00`.
+Um horário possui o formato `hours:minutes`. Ambas as horas e os minutos possuem dois dígitos, como `09:00`.
 
-Make a regexp to find time in the string: `subject:Breakfast at 09:00 in the room 123:456.`
+Componha uma regex que encontra o horário na string: `subject:Café da manhã as 09:00 no quarto 123:456.`
 
-P.S. In this task there's no need to check time correctness yet, so `25:99` can also be a valid result.
+P.S. Nessa tarefa não é necessário verificar a corretude do horário ainda, então `25:99` também pode ser um resultado válido.
```

Comment:
I did not see the problem on my suggestion, but since we can review in the future about ways of improving, I do not see the need to hold this back, so I will approve it.